wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp

```diff
@@ -16,11 +16,18 @@
 
 using namespace frc;
 
+WPI_IGNORE_DEPRECATED
 DifferentialDrive::DifferentialDrive(MotorController& leftMotor,
                                      MotorController& rightMotor)
-    : m_leftMotor(&leftMotor), m_rightMotor(&rightMotor) {
-  wpi::SendableRegistry::AddChild(this, m_leftMotor);
-  wpi::SendableRegistry::AddChild(this, m_rightMotor);
+    : DifferentialDrive{[&](double output) { leftMotor.Set(output); },
+                        [&](double output) { rightMotor.Set(output); }} {}
+WPI_UNIGNORE_DEPRECATED
+
+DifferentialDrive::DifferentialDrive(std::function<void(double)> leftMotor,
+                                     std::function<void(double)> rightMotor)
+    : m_leftMotor{std::move(leftMotor)}, m_rightMotor{std::move(rightMotor)} {
+  // wpi::SendableRegistry::AddChild(this, m_leftMotor);
```

Comment:
I'm not sure what to do about these.

Reply:
Yeah... I forgot about that.  It's not going to be possible to do that with lambdas.  The purpose of this was to nest the motors under the drive base in the Sendable NT tree.  The user could still do this on the calling side, it just wouldn't be automatic.